### PR TITLE
Proposition : A bit more style

### PR DESCRIPTION
**Pull request by bcampardou**
This pull request is a proposition of style using themify icons.
I also added controls on the video tag (when playing a video) to allow fullscreen (and others controls).

capture1
capture2

Let me know what you think about it.

Thanks for your work.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,12 +10,22 @@
...
+  "devDependencies": {
+    "electron-packager": "^10.1.0",
+    "electron-prebuilt": "^1.4.13",
+    "video.js": "^6.4.0"
+  },
   "dependencies": {
     "domify": "^1.4.0",
-    "electron": "1.7.9"
+    "electron": "1.7.9",
+    "menubar": "^5.2.3"
```
Comment (hokein) [inline package.json]: Do we need the menubar here? I couldn't find any usage of your code using it.

**Review by hokein — COMMENTED** [review]
@BaobabCoder 
Awesome! It is really nice, I like the new UI style. Thanks for doing it!
The UI part looks good.

I'm not quite understand the purpose of using `video.js`,  you mentioned that `allow fullscreen for the video`, but the video played still inside the window, and I can't see any difference between using video.js and not using it.

**Comment by bcampardou** [comment]
@hokein You are right for video.js. I just removed it.
For menubar it is a mistake (could be a next PR) you can clone my `menubar` branch to see what it looks like

**Comment by hokein** [comment]
LGTM, Thanks @BaobabCoder!

**Pull request merged**